client/state/selectors/get-checklist-task-urls.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { find, get, some } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getPostsForQuery } from 'state/posts/selectors';
+import getEditorUrl from 'state/selectors/get-editor-url';
+import { getSiteOption } from 'state/sites/selectors';
+import createSelector from 'lib/create-selector';
+
+export const FIRST_TEN_SITE_POSTS_QUERY = { type: 'any', number: 10, order_by: 'ID', order: 'ASC' };
```

Comment:
Is it worth putting this in a constants file somewhere?

Reply:
We could, but I'm not totally sure where to put it! The oddness here is that the query is used as a prop of `QueryPosts` which the component using this selector has to include in a render. In that sense, it seems like a good idea to keep it as an export here. I can't see any other query constants like this in use in the codebase either.

I'd like to see us adopting the [@wordpress/data approach to fetching from the API](https://developer.wordpress.org/block-editor/packages/packages-data/#resolvers). I think using resolvers would make for tidier components.